lib/vanagon/platform/windows.rb

```diff
@@ -107,8 +109,9 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/#{project.name}/staging",
```

Comment:
Have cherry-picked your change 620db16b2b5414fdd008784de4ffc9bcea35a187 into the head of my changelist.
Notice that your change hasn't been merged up yet though.